src/utils/daos_hdlr.c

```diff
@@ -53,6 +54,7 @@ struct file_dfs {
 	daos_off_t offset;
 	dfs_obj_t *obj;
 	dfs_t *dfs;
```

Comment:
if you are switching to the dfs_sys api, is dfs_t *dfs, still needed here?

Reply:
@mchaarawi no, removed it.